schemas/common_definitions.json

```diff
@@ -474,6 +474,10 @@
         "value"
       ]
     }
+  },
+  "url": {
+    "type": "string",
+    "pattern": "^(http:\/\/www\\.|https:\/\/www\\.|http:\/\/|https:\/\/)?[a-z0-9]+([\\-\\.]{1}[a-z0-9]+)*\\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$"
```

Comment:
This seems really complex. Could we get away with something simpler like `http[s]?://\w+`?

Reply:
I've simplified it somewhat. Regex taken from https://www.regextester.com/94502